Remotion/Web/Development.WebTesting/WebTestHelper.cs

```diff
@@ -350,6 +353,49 @@ private void EnsureCursorIsOutsideBrowserWindow ()
       Cursor.Position = new Point (0, 0);
     }
 
+    private string GetBrowserVersion ()
+    {
+      if (!(_mainBrowserSession.Driver.Native is IHasCapabilities driver))
+        return null;
+
+      if (_browserConfiguration.IsChromium())
+        return driver.Capabilities.GetCapability ("version") as string;
+
+      if (_browserConfiguration.IsFirefox())
+        return driver.Capabilities.GetCapability ("browserVersion") as string;
+
+      return null;
+    }
+
+    private string GetWebdriverVersion ()
+    {
+      if (!(_mainBrowserSession.Driver.Native is IHasCapabilities driver))
+        return null;

```

Comment:
Why did you choose to introduce null into the control flow instead of return "unknown" right here?

Reply:
I was thinking about the very same issue while writing the code because I was not sure, but ended up using the `null`-way. The reasoning for this was in order to allow the caller of the respective method(s) to decide what to do with such a result using `null` to indicate this "special" situation.

Reply:
Alright, I see. I don't think it matters much in this case anyway, since the method is private and easily understandable to begin with 👍

Reply:
Fun fact. I just had a siilimar situation with MG. There it was the ProcessWrapper's Name, etc properties. I went with returning a string value all the time since a) there was no benefic to detecting if there's a version and b) it made the API non-null and thus removed complexity on the caller site.

So, here, I guess the same rules apply too. One way, we can just use the API to get a representative string value, the other way, we have to add extra code at the call site. And yes, if we wanted to add semantics to the information about there being a version, things would be very different, designwise.

Reply:
I agree with your reasoning, especially due to the fact that  - as you say - there is no behavior tied to version information being present or not. I have updated the methods to make them return a string value instead of `null`.